stable/traefik/templates/configmap.yaml

```diff
@@ -74,6 +74,10 @@ data:
     {{- if .Values.acme.logging }}
     acmeLogging = true
     {{- end }}
+    {{- if eq .Values.acme.httpChallenge.entryPoint "http" }}
```

Comment:
I'm not sure about this.

Reply:
If you specify [acme.httpChallenge], then you also need to specify the name of the entrypoint. Usually most will use 'http' however a name like 'httpxyz' would be perfectly possible too. So if you start using httpChallenge, you could check a second parameter with the name of the entrypoint. Having said that I did not see any examples use a name other than 'http'.

Reply:
Ah okay, good to know. I'll change it.